src/Umbraco.Web.UI.Client/src/views/prevalueeditors/sortByType.html

```diff
@@ -0,0 +1,6 @@
+<select ng-model="model.value" required>
+  <option value="name-asc">Name ascending</option>
+  <option value="name-desc">Name descending</option>
+  <option value="updateDate-asc">Update date ascending</option>
+  <option value="updateDate-desc">Update date descending</option>
+</select>
```

Comment:
I think this should be more generic to customize property to sort similar to this in listview.

![image](https://github.com/umbraco/Umbraco-CMS/assets/2919859/d783e4a5-f88f-407f-a5f0-07a7011d8f2a)

and something that can be shared across different property editors.

and the direction like this in listview as well.

![image](https://github.com/umbraco/Umbraco-CMS/assets/2919859/de1d034d-56af-4cb4-ac4f-100b5441c454)